paver.sh

```diff
@@ -98,7 +98,13 @@ clean_build() {
     fi
     # In some case pip hangs with a build folder in temp and
     # will not continue until it is manually removed.
-    rm -rf /tmp/pip*
+    # On the OSX build server tmp is in $TMPDIR
+    if [ ! -z "${TMPDIR-}" ]; then
```

Comment:
actually for some reason `-n` would give us a false positive:

```
[bgola@charango noisegrid]$ echo ${xx}
bash: xx: unbound variable
[bgola@charango noisegrid]$ if [ ! -z ${xx-} ]; then echo "not set"; fi
[bgola@charango noisegrid]$ if [ -n  ${xx-} ]; then echo "not set"; fi
S
```

so I will keep `! -z`